src/Symfony/Component/DependencyInjection/Loader/YamlFileLoader.php

```diff
@@ -22,6 +22,7 @@
 use Symfony\Component\Yaml\Exception\ParseException;
 use Symfony\Component\Yaml\Parser as YamlParser;
 use Symfony\Component\ExpressionLanguage\Expression;
+use Symfony\Component\Yaml\Yaml;
```

Comment:
Are you sure about this? The "good way" would be to move up the `Expression` line but as a CS fix it shouldn't be done in master, right?